src/test/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/ObjectionServiceTest.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.companieshouse.api.strikeoffobjections.service.impl;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.companieshouse.api.strikeoffobjections.common.ApiLogger;
+import uk.gov.companieshouse.api.strikeoffobjections.model.entity.StrikeOffObjectionsEntity;
+import uk.gov.companieshouse.api.strikeoffobjections.repository.StrikeOffObjectionsRepository;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+import static org.mockito.ArgumentMatchers.any;
+
+@ExtendWith(MockitoExtension.class)
+class ObjectionServiceTest {
+
+    private static final String COMPANY_NUMBER = "12345678";
+    private static final String REQUEST_ID = "87654321";
+    private static final String OBJECTION_ID = "87651234";
+
+    @Mock
+    ApiLogger apiLogger;
+
+    @Mock
+    StrikeOffObjectionsRepository strikeOffObjectionsRepository;
+
+    @InjectMocks
+    ObjectionService objectionService;
+
+    @Test
+    void createObjectionTest() {
+        StrikeOffObjectionsEntity returnedEntity = new StrikeOffObjectionsEntity.Builder()
+                .withCompanyNumber(COMPANY_NUMBER)
+                .build();
+        returnedEntity.setId(OBJECTION_ID);
+        when(strikeOffObjectionsRepository.save(any())).thenReturn(returnedEntity);
+
```

Comment:
could you mock the builder and check it is called with the correct company number and current timestamp (supplier?)
Then you could confirm that the entity returned from the builder is the same one passed into the save() call

Reply:
I thought as the builder is static, Mockito cannot mock it?

Reply:
ok, but we need a way to check the entity is created with correct values. If we can't mock the builder, let it build a real one, capture the argument passed in to save() and check it has the same company number that was passed in to the method and that the timestamp is correct. 
You could still have a dummy response returned from save() and check the id is being returned from createObjection (as it's currently doing)